File: Sum of Array/Sum_of_Array.java
class Solution {
    int sum(int arr[]) {
        // simple question 
        // we have to just add all element 
        // we can do this with linear traversal
        // linear traversal means visit every element
        // one by one and add into total
        int total = 0;
        int n = arr.length;
        for (int i = 0; i < n; i++) {
            total += arr[i];
        }
        return total;
    }
}

File: Sum of Array/Sum_of_Array.py
class Solution:
	def _sum(self,arr):
   		# simple question 
        # we have to just add all element 
        # we can do this with linear traversal
        # linear traversal means visit every element
        # one by one and add into total
        total = 0
        n = len(arr)
        for i in range(n):
            total += arr[i]
        return total
